### PR TITLE
Holberton Hackathon

### PR DESCRIPTION
**Pull request by sidneyriffic**
I left in a few models of my models backup folders that are a bit farther along in epochs that don't quite have as much loss if you wanted to look at them. The one I put into "weights" was the lowest for my most recent model, and is the same as the 2-24 folder.

### CODE REVIEW

**Comment by GiorgioRegni** [comment]
Thank you Sidney!

**Comment by sidneyriffic** [comment]
The model just started spitting out a bunch of stuff with half the loss just now, heh. A quick push for that. It's in weights/the 5-13 folder.

**Comment by GiorgioRegni** [comment]
The juicy parts :) :
`units= CONFIG['lstm_hidden_size']
dropout = .1

# design network
inputs = Input(shape=(48, 4))
firstconv = Conv1D(30, 3)(inputs)
firstconv = LSTM(30)(firstconv)
firstconv = Dropout(dropout)(firstconv)
firstrnn = LSTM(30)(inputs)
firstrnn = Dropout(dropout)(firstrnn)
firstcat = concatenate([firstconv, firstrnn])
results = Dense(60)(firstcat)
results = Dropout(dropout)(results)
results = Dense(4)(results)
results = keras.layers.Reshape((1, 4))(results)

innerputs = concatenate([inputs, results], axis=1)
for i in range(1, 12):
    
    innerconv = Conv1D(30, 3)(innerputs)
    innerconv = LSTM(30)(innerconv)
    innerconv = Dropout(dropout)(innerconv)
    innerrnn = LSTM(30, input_shape=(48 + i, 4))(innerputs)
    innerrnn = Dropout(dropout)(innerrnn)
    innercat = concatenate([innerconv, innerrnn])
    innerres = Dense(60)(innercat)
    innerres = Dropout(dropout)(innerres)
    innerres = Dense(4)(innerres)
    innerres = keras.layers.Reshape((1, 4))(innerres)
    results = concatenate([results, innerres], axis = 1)
    innerputs = concatenate([innerputs, innerres], axis=1)`

**Comment by sidneyriffic** [comment]
Yeah, that is the best part. It was pretty cool to think up. But I at least still need to make sure it's doing something like what I want it to or at least think it might do! Which it seems to be, fortunately.
